Minor correction to FCM Push Notifications Guide (#21370)
supabase/supabase: apps/docs/content/guides/functions/examples/push-notifications.mdx

    We will create two tables. One to store the user's FCM token and a `notifications` table. The edge function will be triggered when a new row is inserted into the `notifications` table and sends a push notification to the user.

>     Run the following SQL to create the tables if you don't already have both tables.

Reviewer: ```suggestion
    Create a `notifications` table. Also create a `profiles` table if you don't already have one:
```

      id uuid not null default gen_random_uuid(),
      user_id uuid references auth.users(id) not null,
      created_at timestamp with time zone not null default now(),
      body text not null
    );
    ```

>     You may have to alter an existing profiles table. For example, if you previously followed a separate Supabase [Build A User Management](https://supabase.com/docs/guides/getting-started/tutorials/with-flutter) guide which added a `profiles` table. Then you need the following SQL for `public.profiles`:

Reviewer: ```suggestion
    If you already have a `profiles` table, alter it to include an `fcm_token` column:
```

    ```sql
    ALTER TABLE public.profiles 

>     ADD COLUMN  fcm_token text;

Reviewer: ```suggestion
    ADD COLUMN fcm_token text;
```
